taskvine/src/bindings/python3/ndcctools/taskvine/task.py

```diff
@@ -12,15 +12,20 @@
 from .file import File
 
 import copy
-import json
 import os
 import shutil
 import sys
 import tempfile
 import textwrap
 import uuid
 import weakref
-
+try:
+    import cloudpickle
```

Comment:
So, this pythontask_available runaround was due to missing dependencies not in the standard Conda package.

But we can easily add packages to the conda package definition, as well as our developer dependencies.  Is there a strong reason to make cloud pickle optional instead of required?

Reply:
The only reason I can think of is somebody who only wants the C core of TaskVine, which makes cloudpickle redundant.
Since we want users to use `ndcctools` via conda install, do you know if `cloudpickle` is already in ndcctools dependencies?
@btovar What do you think about this?

Reply:
I suppose all of the other imports are python standard library.  I don't have a strong feeling about this, just want to make sure you are proceeding deliberately...

Reply:
I think we should make it required. It is installed as a conda dependency. The C code parts should work even if you don't have python installed.